app/src/pages/header/components/AddMovieForm.test.js

```diff
@@ -0,0 +1,174 @@
+import React from 'react';
+import { AddMovieForm } from './AddMovieForm.jsx';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { Movie } from '@models';
+
+it('AddMovieForm UI testing', () => {
+    render(<AddMovieForm />);
+
+    let header = screen.queryByText('ADD MOVIE')
+    expect(header).toBeInTheDocument();
+
+    let titleLabel = screen.queryByText('TITLE')
+    expect(titleLabel).toBeInTheDocument();
+    let titleInput = screen.getByTitle('title');
+    expect(titleInput).toBeInTheDocument();
+
+    let releaseDateLabel = screen.queryByText('RELEASE DATE')
+    expect(releaseDateLabel).toBeInTheDocument();
+    let releaseDateInput = screen.getByTitle('release_date');
+    expect(releaseDateInput).toBeInTheDocument();
+
+    let posterPathLabel = screen.queryByText('MOVIE URL')
+    expect(posterPathLabel).toBeInTheDocument();
+    let posterPathInput = screen.getByTitle('poster_path');
+    expect(posterPathInput).toBeInTheDocument();
+
+    let genresLabel = screen.queryByText('GENRES')
+    expect(genresLabel).toBeInTheDocument();
+    let genresInput = screen.getByTitle('genres');
+    expect(genresInput).toBeInTheDocument();
+
+    let overviewLabel = screen.queryByText('OVERVIEW')
+    expect(overviewLabel).toBeInTheDocument();
+    let overviewInput = screen.getByTitle('overview');
+    expect(overviewInput).toBeInTheDocument();
+
+    let runtimeLabel = screen.queryByText('RUNTIME')
+    expect(runtimeLabel).toBeInTheDocument();
+    let runtimeInput = screen.getByTitle('runtime');
+    expect(runtimeInput).toBeInTheDocument();
+
+    let submitButton = screen.queryByText('SUBMIT')
+    expect(submitButton).toBeInTheDocument();
+
+    let resetButton = screen.queryByText('RESET')
+    expect(resetButton).toBeInTheDocument();

```

Comment:
u could use snapshots instead of each .toBeInTheDocument() call

Reply:
Great idea, thank you! Done